server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -71,6 +73,13 @@ export class ScriptoriumLambda implements IPartitionLambda {
             return;
         }
 
+        let metric: Lumber<LumberEventName.ScriptoriumHandler>;
+        if (this.telemetryEnabled) {
+            metric = Lumberjack.newLumberMetric(
+                LumberEventName.ScriptoriumHandler,
```

Comment:
This metric seems to be specific to track the `handler()` method, but is applied to the `sendPending()` subroutine online (that is, parts of `handler()` are not captured in the metric. Was that intentional? Would it make sense to either rename the metric name, or move it instead to the main `handler()` method?

Reply:
I am tracking it in `sendPending` instead of `handler` since `handler`'s logs might occur too frequently. I can change the metric name to avoid this confusion. Thanks for the feedback :)